runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerServiceImpl.java

```diff
@@ -57,6 +57,12 @@ public int getActionsLeft(Skill skill)
 		return plugin.getSkillSnapshot(skill).getActionsRemainingToGoal();
 	}
 
+	@Override
+	public int getKillsLeft(Skill skill)
+	{
+		return plugin.getXpPanel().getInfoBoxes().get(skill).getKillsRemaining(plugin.getSkillSnapshot(skill));
```

Comment:
Ugh, just make a function on the plugin to actually get kills remaining instead of throwing this crapton of nesting getters here. What if any of them is null in the future? This is fragile